atom/browser/api/atom_api_window.cc

```diff
@@ -858,6 +864,7 @@ void Window::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("unhookWindowMessage", &Window::UnhookWindowMessage)
       .SetMethod("unhookAllWindowMessages", &Window::UnhookAllWindowMessages)
       .SetMethod("setThumbnailClip", &Window::SetThumbnailClip)
+      .SetMethod("setThumbnailTooltip", &Window::SetThumbnailTooltip)
```

Comment:
I guess it's too late... I haven't read his comment properly :)
Renamed to `setThumbnailToolTip`, even though I don't like it much, as I think tooltip is a single word.